content/My favorite extensions for programming Ruby in Visual Studio Code.md

```diff
@@ -0,0 +1,106 @@
+# Choosing a text editor
```

Comment:
I think maybe all this `#` you have should be `##` like subtitles and you should have one main title `My favorite Visual Studio Code extension for Ruby language` or something like that. WDYT? Because I felt like I started already with a `Choosing a text editor` subtitle and no title for the article.

Reply:
Seems like this is the actual title of the post, right ?

Maybe it should be what Manu suggested or similar:

```
# My favorite Visual Studio Code extensions for Ruby
```